packages/select/src/vaadin-select.js

```diff
@@ -5,6 +5,8 @@
  */
 import '@polymer/iron-media-query/iron-media-query.js';
 import '@vaadin/input-container/src/vaadin-input-container.js';
+import './vaadin-select-item.js';
+import './vaadin-select-list-box.js';
```

Comment:
Oh, one more thing. The JSdocs say `"The content of the select can be populated by using the renderer callback function."`.

Should update the documentation to promote `items` as the primary means for defining the select options.

Reply:
Added. Please check it out.